docker-compose.yml

```diff
@@ -20,14 +17,11 @@ services:
       - TXTL_LOG_FILE
     ports:
       - "127.0.0.1:3006:3006"
+      - "3007:3007"
       - "4006:4006"
-      - "5006:5006"
-      - "5007:5007"
-      - "6006:6006"
-      - "6007:6007"
       - "127.0.0.1:8006:8006"
   ipfs:
-    image: ipfs/go-ipfs:latest
+    image: ipfs/go-ipfs:v0.5.0-rc2
     restart: on-failure
     volumes:
```

Comment:
For completeness might be good to add MongoDB as a dep to be spinned.

Reply:
Yep, nice. Added.